docs/zh_cn/notes/faq.md

```diff
@@ -172,7 +172,7 @@ PYTHONPATH="$(dirname $0)/..":$PYTHONPATH
 
 - 训练中保存最好模型
 
-  可以通过配置 `evaluation = dict(save_best=‘auto’)`开启。在 auto 参数情况下会根据返回的验证结果中的第一个 key 作为选择最优模型的依据，你也可以直接设置评估结果中的 key 来手动设置，例如 `evaluation = dict(save_best=‘mAP’)`。
+  可以通过配置 `default_hooks = dict(checkpoint=dict(type='CheckpointHook', interval=1, save_best='auto')`开启。在 auto 参数情况下会根据返回的验证结果中的第一个 key 作为选择最优模型的依据，你也可以直接设置评估结果中的 key 来手动设置，例如 `save_best='mAP'`。
```

Comment:
```suggestion
  可以通过配置 `default_hooks = dict(checkpoint=dict(type='CheckpointHook', interval=1, save_best='auto')`开启。在 `auto` 参数情况下会根据返回的验证结果中的第一个 key 作为选择最优模型的依据，你也可以直接设置评估结果中的 key 来手动设置，例如 `save_best='mAP'`。
```

Reply:
Also need to replace `save_best='mAP'` to `save_best='coco/bbox_mAP'` in the English doc.